.github/workflows/build-timestamped-master.yml

```diff
@@ -36,12 +36,5 @@ jobs:
           publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
           ./gradlew publish --scan --no-daemon
-      - name: Archive Error Log
```

Comment:
Any reason to remove this? This was added so as if any ballerina-internal-log is produced, we can check this without reproducing the issue locally.

Reply:
With the recent changes, there's no more `internal.log` file created. The log will be printed to the stderr stream.

Reply:
Ack. Lets add this as a step in the migration.

Reply:
Added in the issue.